### DISCUSSION TITLE
Using Apollo client with Angular on Android

### DISCUSSION THREAD

**Discussion by TomasMoskala** [prisma/prisma#2912] [Q&A] [answered]
Hi, 
is there any example of using Apollo client with Angular on Android? It works for me with ionic serve on PC but using capacitor to create an Android project I cannot get any response from Apollo.

**Comment by janpio** [comment]
Are you using Prisma in your project?
Or Nexus?

**Reply by TomasMoskala** [reply]
Hi, 
I am using Prisma on Apollo server. The client app is not using it.

**Reply by ryands17** [reply]
Hey @TomasMoskala :wave: 
Currently we do not have any examples of Prisma with Apollo Client + Angular so could you open an example request [here](https://github.com/prisma/prisma-examples/issues/new) so that we can look into it.

**Reply by TomasMoskala** [reply]
Hi, 
I have the example ready I'm just thinking that the issue might not be with Prisma but with Apollo, because I've tried to connect to the public SpaceX API https://api.spacex.land/graphql/ and my app works. I can display data in both Android studio simulator and on device.
Do you guys still have time to look into it?

**Reply by TomasMoskala** [reply]
Hi, 
so I made a repo here
https://github.com/TomasMoskala/angular-apollo-app
The server palyground should be live at http://test-4233.rostiapp.cz/
It still does not work so any help is highly appreciated.

**Comment by TomasMoskala** [comment, marked as answer]
Solved

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
